examples/publish_hue.py

```diff
@@ -38,8 +39,14 @@ async def main(room: rtc.Room):
     # publish a track
     source = rtc.VideoSource(WIDTH, HEIGHT)
     track = rtc.LocalVideoTrack.create_video_track("hue", source)
-    options = rtc.TrackPublishOptions()
-    options.source = rtc.TrackSource.SOURCE_CAMERA
+    options = rtc.TrackPublishOptions(
+        source=rtc.TrackSource.SOURCE_CAMERA,
+        simulcast=True,
+        video_encoding=rtc.VideoEncoding(
+            max_framerate=FPS,
```

Comment:
max_framerate might need to be a float here

Reply:
we should convert automatically? because that's non-obvious to a user

Reply:
python should do it automatically?